# HG changeset patch
# User Andrew John Hughes <gnu.andrew@redhat.com>
# Date 1390438416 0
#      Thu Jan 23 00:53:36 2014 +0000
# Node ID f49a6db8092f35b9f12af481c81ffb2aab8c3fdd
# Parent  f1cb7c2a533bb623be76974ef3d9d34a778c6bb2
Start 1.13.2 release cycle.

2014-01-23  Andrew John Hughes  <gnu.andrew@redhat.com>

	* configure.ac: Set to 1.13.2pre.
	* NEWS: Add 1.13.2 section and missing CVE.

diff -r f1cb7c2a533b -r f49a6db8092f ChangeLog
--- a/ChangeLog	Wed Jan 22 15:07:55 2014 +0000
+++ b/ChangeLog	Thu Jan 23 00:53:36 2014 +0000
@@ -1,3 +1,8 @@
+2014-01-23  Andrew John Hughes  <gnu.andrew@redhat.com>
+
+	* configure.ac: Set to 1.13.2pre.
+	* NEWS: Add 1.13.2 section and missing CVE.
+
 2014-01-22  Andrew John Hughes  <gnu.andrew@redhat.com>
 
 	* configure.ac: Set to 1.13.1.
diff -r f1cb7c2a533b -r f49a6db8092f NEWS
--- a/NEWS	Wed Jan 22 15:07:55 2014 +0000
+++ b/NEWS	Thu Jan 23 00:53:36 2014 +0000
@@ -12,6 +12,8 @@
 
 CVE-XXXX-YYYY: http://www.cve.mitre.org/cgi-bin/cvename.cgi?name=XXXX-YYYY
 
+New in release 1.13.2 (2014-04-XX):
+
 New in release 1.13.1 (2014-01-22):
 
 * Security fixes
@@ -20,7 +22,7 @@
   - S8010935: Better XML handling
   - S8011786, CVE-2014-0368: Better applet networking
   - S8021257, CVE-2013-5896: com.sun.corba.se.** should be on restricted package list
-  - S8021271: Better buffering in ObjC code
+  - S8021271, CVE-2014-0408: Better buffering in ObjC code
   - S8022904: Enhance JDBC Parsers
   - S8022927: Input validation for byte/endian conversions
   - S8022935: Enhance Apache resolver classes
diff -r f1cb7c2a533b -r f49a6db8092f configure.ac
--- a/configure.ac	Wed Jan 22 15:07:55 2014 +0000
+++ b/configure.ac	Thu Jan 23 00:53:36 2014 +0000
@@ -1,4 +1,4 @@
-AC_INIT([icedtea6],[1.13.1],[distro-pkg-dev@openjdk.java.net])
+AC_INIT([icedtea6],[1.13.2pre],[distro-pkg-dev@openjdk.java.net])
 AC_CANONICAL_HOST
 AC_CANONICAL_TARGET
 AM_INIT_AUTOMAKE([1.9 tar-pax foreign])
